suite-common/suite-types/src/guide.tsx

```diff
@@ -1,7 +1,37 @@
-import { Page, GuideCategory, Node } from '@trezor/suite-data/src/guide/parser';
-
 import type { EnvironmentType } from './environment';
 
+/**
+ * A group of Guide content.
+ * Can contain other Pages or Categories.
+ * Cannot contain content on its own except the title.
+ */
+export interface GuideCategory {
+    type: 'category';
+    /** Serves both as unique identifier and relative path to the directory. */
+    id: string;
+    /** List of locales this Category is available in. */
+    locales: string[];
+    /** Titles keyed by locales. */
+    title: {
+        [key: string]: string;
+    };
+    image?: string;
+    /** Sub-categories and sub-pages. */
+    children: Node[];
+}
+
+/** A single unit of Guide content. */
+export interface Page {
+    type: 'page';
+    id: string;
+    locales: string[];
+    title: {
+        [key: string]: string;
+    };
+}
+
+export type Node = GuideCategory | Page;
```

Comment:
Now when the Guide types are imported directly from `@suite-common/suite-types`, I would rename them, probably to `GuideNode` and `GuidePage`. Without it, `import { Node } from '@suite-common/suite-types'` is kind of confusing.

Reply:
0a6082ba8edd0e0f28178ca058948e33e63ead0f